site/src/App/foundations/layout/layout.tsx

```diff
@@ -0,0 +1,512 @@
+          <Bullet>paddingRight</Bullet>
+        </BulletList>
+        <Text>These options accept a value from our white space scale.</Text>
+        <Text>
+          For example, if you wanted to create a container element with small
+          spacing on all sides
+        </Text>
+        <Code>
+          <Box padding="small">
+            <Text>Lorem ipsum dolor sit amet.</Text>
+          </Box>
+        </Code>
+        <Text>
+          These also support the ‘responsive props’ format which allows you to
+          specify an array of values for different screen sizes.
+        </Text>
+        <Text>
+          For example, if you wanted small spacing on mobile but medium spacing
+          from tablet upwards:
+        </Text>
+        <Code>
+          <Box padding={['small', 'medium']}>
+            <Text>Lorem ipsum dolor sit amet.</Text>
+          </Box>
+        </Code>
+        <Text>
+          If required, you’re also able to specify a different value for desktop
+          screens.
+        </Text>
+        <Text>
+          For example, if you wanted to set the previous example’s spacing to
+          large on desktop:
+        </Text>
+        <Code>
+          <Box padding={['small', 'medium', 'large']}>
+            <Text>Lorem ipsum dolor sit amet.</Text>
+          </Box>
+        </Code>
+
+        <Divider />
+
+        <Heading level="3">Card</Heading>
+        <Text>
+          Rather than nesting content in arbitrary Box elements, you may prefer
+          to use standard Card elements instead.
+        </Text>
+        <Code>
+          <Card>
+            <Text>Lorem ipsum dolor sit amet.</Text>
+          </Card>
+        </Code>
+
+        <Divider />
+
+        <Heading level="3">Stack</Heading>
+        <Text>
+          The most common white space on screen is between elements stacked
+          vertically. For this use case, Braid provides a Stack component that
+          accepts a ‘space’ option.
+        </Text>
+        <Text>
+          For example, if you wanted to render a stack of Heading and Text
+          elements with large spacing between them:
+        </Text>
+        <Code>
+          <Card>
+            <Stack space="large">
+              <Heading level="3">Heading</Heading>
+              <Text>{lipsum1}</Text>
+              <Text>{lipsum2}</Text>
+            </Stack>
+          </Card>
+        </Code>
+        <Text>
+          Just like Box, you can also specify different spacing values for
+          different screen sizes:
+        </Text>
+        <Code>
+          <Card>
+            <Stack space={['medium', 'large']}>
+              <Heading level="3">Heading</Heading>
+              <Text>{lipsum1}</Text>
+              <Text>{lipsum2}</Text>
+            </Stack>
+          </Card>
+        </Code>
+        <Text>
+          To visually break up content, you can insert dividers between all
+          stack elements by setting the ‘dividers’ prop on Stack:
```

Comment:
As a consistency point, should we use `prop` or `option`? Happy for either but should probably standardise.